randovania/games/prime2/json_data/Torvus Bog.txt

```diff
@@ -776,7 +776,7 @@ Extra - in_dark_aether: True
                   Dark World Damage ≥ 40
                   Any of the following:
                       Before Dark Forgotten Bridge Rotated
-                      Screw Attack and Movement (Beginner)
+                      Movement (Beginner) and Use Screw Attack (No Space Jump)
```

Comment:
Morph is needed to unlock the gate in Forgotten Bridge, so listing it here is redundant